### ISSUE TITLE
Invalid password

### ISSUE THREAD

**Issue by Quax456** [eracknaphobia/plugin.video.mlbtv#17]
I got an error everytime I try to watch a game withe invalid password.

Have checked twice possible typos.

**Comment by eracknaphobia** [comment]
I suspect I need to escape special characters within the password field. By chance does your password contain any of these characters(<, >, &, " or ')?

**Comment by Quax456** [comment]
No. My Password contains only Lower  and upper Cause and Nummer.

**Comment by Quax456** [comment]
i have to appologize. hd a typo in my password. I am so stupid. had to check it twice before moaning.

thx for this great plugin!
